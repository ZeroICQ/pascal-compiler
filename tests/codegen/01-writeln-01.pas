 begin
    writeln('hello,');
    writeln('world');
 end.