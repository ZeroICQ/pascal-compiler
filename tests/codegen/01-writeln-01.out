hello

world!2020
20
1.2222222199999999E+000
1.2330000000000000E-010
